components/mosart/cime_config/testdefs/testmods_dirs/mosart/sediment/user_nl_mosart

```diff
@@ -0,0 +1,5 @@
+frivinp_rtm = '/compyfs/inputdata/rof/mosart/MOSART_NLDAS_8th_20160426_20210609d_rslp.nc'
```

Comment:
@liho745 Use `$DIN_LOC_ROOT` instead of hard-coding the path

Reply:
Done